sherpa/python/csrc/sherpa.cc

```diff
@@ -19,15 +19,15 @@
 #include "sherpa/python/csrc/sherpa.h"
 
 #include "sherpa/python/csrc/rnnt_beam_search.h"
+#include "sherpa/python/csrc/rnnt_conformer_model.h"
 #include "sherpa/python/csrc/rnnt_emformer_model.h"
-#include "sherpa/python/csrc/rnnt_model.h"
 
 namespace sherpa {
 
 PYBIND11_MODULE(_sherpa, m) {
   m.doc() = "pybind11 binding of sherpa";
 
-  PybindRnntModel(m);
+  PybindRnntConformerModel(m);
```

Comment:
please refer https://pybind11.readthedocs.io/en/stable/advanced/classes.html#combining-virtual-functions-and-inheritance
to tell pybind11 and python about the inheritance relationship among RnntModel, RnntEmformermodel and RnntConformerModel.

You can use
https://github.com/kaldi-asr/kaldi/blob/pybind11/src/pybind/matrix/kaldi_matrix_pybind.cc
as a reference.

Reply:
Please fix this issue.

Reply:
> Please fix this issue.

I thought I have fixed it, could you point it out more clearly, thanks!

Reply:
1.  You have not wrapped `RnntModel` to Python.
2. The `RnntEmformermodel` in Python does don't inherit from `RnntModel`.